src/datadog_api_client/configuration.py

```diff
@@ -344,6 +344,9 @@ def __init__(
             }
         )
 
+        # Delegated token configuration
+        self.delegated_token_config = None
+
         # Load default values from environment
```

Comment:
Looks like constructors are missing in the config for fields such as `delegated_auth_provider` and `delegated_auth_org_uuid`